crates/runix/src/command.rs

```diff
@@ -227,3 +228,16 @@ impl JsonCommand for Bundle {}
 impl TypedCommand for Bundle {
     type Output = ();
 }
+
+/// `nix store gc` Command
+#[derive(Debug, Default, Clone)]
+pub struct StoreGc {
+    pub store_gc: StoreGcArgs,
+}
+
+impl NixCliCommand for StoreGc {
+    type Own = StoreGcArgs;
+
+    const OWN_ARGS: Group<Self, Self::Own> = Some(|d| d.store_gc.clone());
```

Comment:
Let's add a comment that `store gc` does not support any of the other argument Sets, this looks like something is missing..